src/plugins/data/public/search/session/sessions_client.ts

```diff
@@ -92,6 +92,10 @@ export class SessionsClient {
     });
   }
 
+  public cancel(sessionId: string): Promise<void> {
+    return this.http!.post(`/internal/session/${encodeURIComponent(sessionId)}/cancel`);
```

Comment:
Super minor nit: Can we either standardize on `/_action` or `/action`? I don't really have a preference (though saved objects are `/_action`), just want them to be the same format.

Reply:
To me, it would make more sense to swallow the 404 errors here in the client rather in the API itself, and only log the errors on the server if it's not a 404. What do you think?

Reply:
I didn't intend do to this first, but you can't eliminate the 404 from the console, and it looks bad IMO
https://stackoverflow.com/questions/12915582/eliminate-404-url-error-in-console

What do you think?